features_and_tools/multi-tenancy.rst

```diff
@@ -52,14 +58,35 @@ You can control the working of the feature with the settings:
             "TenantClaim": "tenant",
             "TenantLabelSystem": "http://server.fire.ly/fhir/sid/tenant-label",
             "AllowTenantFromHeader": true,
-            "AllowTenantFromClaim": true
+            "AllowTenantFromClaim": true,
+            "RemoveTenantLabelFromResponse": true,
+            "ResourceAccess": {
+                "<unique-key>": {
+                  "Priority": 100,
+                  "Tenant": "Required"
+                },
+                "<unique-key>": {
+                  "Priority": 200,
+                  "Tenant": "Shared"
+                  "Profiles": [],
+                  "ResourceTypes": ["Patient", "Practitioner"],
+                  "Filter": "_tag=vip"
+                }
+            }
         },
 
 :TenantHeader: The name of the HTTP header to read the tenant from. This is only evaluated if ``AllowTenantFromHeader`` is set to ``true``.
 :TenantClaim: The name of the claim in the authorization token to read the tenant from. This is only evaluated if ``AllowTenantFromClaim`` is set to ``true``.
 :TenantLabelSystem: The value of the ``meta.security.system`` element to use for the tenant security label. You can only choose this once.
 :AllowTenantFromHeader: The tenant may be specified with an HTTP header, with the name specified in ``TenantHeader``.
 :AllowTenantFromClaim: The tenant may be specified with a claim in the authorization token, with the name specified in ``TenantClaim``.
+:RemoveTenantLabelFromResponse: The label will be removed from the resource by default, but this behaviour can be disabled if this option is set to ``false``.
+:ResourceAccess: This structure allows to define set of conditional logic that will be applied to each request.
+:ResourceAccess:Priority: This value will be used to choose most important logic entry if multiple entries would match the request.
+:ResourceAccess:Tenant: This value specifies whether the tenant is required to be specified when accessing the resource. It has to be either ``Required`` or ``Shared``.
+:ResourceAccess:Profiles: This value specifies a list of ``meta.profile`` values to look for in the resource. The resource should have at least one of those values.
+:ResourceAccess:ResourceTypes: This value specifies a list of resources this configuration will be applicable to. When left empty, it is assumed to be not constrained.
```

Comment:
```suggestion
:ResourceAccess:ResourceTypes: This value specifies a list of resource types this configuration will be applicable to. When left empty, it is assumed to be not constrained.
```
- 'resources' and 'resource types' are casually confused, but as FHIR pro's we should not do that. A 'resource' is an instance.
- I assume you update this based on our discussed meaning of an empty array (i.e.: nothing matches).